jupyter-jekyll.tpl

```diff
@@ -54,17 +54,9 @@ In&nbsp;[{{ cell.execution_count }}]:
 {% endblock data_svg %}
 
 {% block data_png %}
-[![](data:image/png;base64,{{ output.data['image/png'] }})]({{ output.metadata.filenames['image/png'] | path2url }})
+![png](data:image/png;base64,{{ output.data['image/png'] }})
```

Comment:
This was causing the images to be links in the final page. I removed that because we don't really want clickable images. I also cleanup some formats we do not use below.